service-worker.js

```diff
@@ -279,9 +294,10 @@ self.addEventListener('fetch', function (event) {
  * Handles URLs that need to be extracted from the ZIM archive
  * 
  * @param {URL} urlObject The URL object to be processed for extraction from the ZIM
+ * @param {String} range optional byte range string
```

Comment:
Just for clarity, I'd make "Optional" with capital letter, as it separates the variable name from the description visually.

Reply:
I agree